.pre-commit-hooks/runtime-versioner.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+cd "$(git rev-parse --show-toplevel)/tools/ci-build/runtime-versioner" && cargo run -- audit
```

Comment:
this runs on every single runtime crate? I guess the tool itself has the crate list?

Reply:
That's correct.